package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "axios": "^1.7.7"
+  }
+}
```

Comment:
Oh whoops, it got added to the root directory 😂

Reply:
thanks for the catch!